doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1442 @@
+
+* Deprecated support for the nRF7002 EB (PCA63561) with the nRF54 Series DKs.
+* Removed support for storing the nRF70 firmware patches in external flash without the :ref:`partition_manager`, as mentioned in :ref:`ug_nrf70_developing_fw_patch_ext_flash`.
+
+Developing with nRF54L Series
+=============================
+
+* Added:
+
+  * HMAC SHA-256 with a 128-bit key type to KMU, as detailed in the :ref:`ug_nrf54l_crypto_kmu_supported_key_types` documentation section.
+  * A workaround for nRF54L15 Errata 30.
+    Use ``CONFIG_CLOCK_CONTROL_NRF_HF_CALIBRATION=y`` to explicitly activate the workaround.
+    A consequence of activating the workaround might be increased power consumption due to periodic CPU wake-up, so use it only if errata conditions are met.
+    The workaround is already included in the MPSL component, so the solution is applicable only if your application does not use MPSL.
+
+Developing with nRF54H Series
+=============================
+
+* Added a new documentation page :ref:`ug_nrf54h20_suit_signing` under :ref:`ug_nrf54h20_suit_dfu`.
+* Removed the note on installing SEGGER USB Driver for J-Link on Windows from the :ref:`ug_nrf54h20_gs` page and moved its contents to the `nRF Util prerequisites`_ documentation.
+  The Windows-only requirement to install the SEGGER USB Driver for J-Link is now mentioned in the :ref:`installing_vsc` section on the :ref:`installation` page.
+
+Developing with Front-End Modules
+=================================
+
+* Added support for the following:
+
+  * :ref:`nRF2220 Front-End Module <ug_radio_fem_nrf2220>`.
+  * :ref:`nRF2220 EK shield <ug_radio_fem_nrf2220ek>`.
+  * :ref:`nRF21540 Front-End Module in GPIO mode <ug_radio_fem_nrf21540_gpio>` for the nRF54L Series devices.
+
+* Fixed an issue for the nRF21540 Front-End Module (for GPIO and GPIO+SPI modes) when spurious emission occurred due to late activation of the ``TX_EN`` pin.
+  The ``PDN`` pin is now activated earlier by the call to the :c:func:`mpsl_fem_enable` function.
+  The ``TX_EN`` pin is now activated 15 µs earlier during the ramp-up of the radio.
+
+Developing with PMICs
+=====================
+
+* Added the :ref:`ug_npm2100_developing` documentation.
+
+Security
+========
+
+* Added:
+
+  * Support for HKDF-Expand and HKDF-Extract in CRACEN.
+  * Support for HashEdDSA (ed25519ph) to CRACEN.
+  * TF-M now supports Attestation service on nRF54L15
+  * The following documentation pages:
+
+    * :ref:`ug_tfm_architecture`.
+    * :ref:`ug_psa_certified_api_overview`.
+    * :ref:`ug_tfm_supported_services`.
+
+* Updated:
+
+  * The Oberon PSA core to version 1.3.4 that introduces support for the following:
+
+    * PSA static key slots with the option :kconfig:option:`CONFIG_MBEDTLS_PSA_STATIC_KEY_SLOTS`.
+    * Key Wrap with and without padding (NIST-SP-800-38F) using Oberon PSA driver.
+    * WPA3-SAE and WPA3-SAE-PT using Oberon PSA driver.
```

Comment:
Let's remove lines mentioning "Key wrap" and "WPA3-SAE".
Those are not standardized as PSA Crypto features yet, and they should not be relied on for external users of the SDK. They are only available for an internal evaluation so far.

Reply:
Removed